Documentation/ceph-csi-snapshot.md

```diff
@@ -6,14 +6,17 @@ indent: true
 
 ## Prerequisites
 
-* Requires Kubernetes v1.17+ which supports snapshot beta.
+#### Snapshot API version support matrix
 
-* Install the new snapshot controller and snapshot beta CRD. More info can be found
-[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage)
+| Snapshot API version | Kubernetes Version   | Snapshot-Controller + CRDs Version | Sidecar Version |
+| -------------------- | -------------------- | ---------------------------------- | --------------- |
+| v1                   | k8s >= v1.20         | snapshot-controller >= v4.x        | sidecar >= v2.x |
+| v1beta1              | v1.17 =< k8s < v1.20 | v2.x =< snapshot-controller < v4.x | sidecar >= v2.x |
+| v1alpha1             | v1.12 =< k8s < v1.17 | snapshot-controller <= v1.x        | sidecar <= v1.x |
+Note: It is recommend to use {sidecar, controller, crds} of same version.
```

Comment:
The rendering is adding this line to the table, you will need a blank line before and after this note.